src/c++/library/grpc_client.h

```diff
@@ -156,15 +156,23 @@ class InferenceServerGrpcClient : public InferenceServerClient {
   /// \param live Returns whether the server is live or not.
   /// \param headers Optional map specifying additional HTTP headers to include
   /// in the metadata of gRPC request.
-  /// \return Error object indicating success or failure of the request.
-  Error IsServerLive(bool* live, const Headers& headers = Headers());
+  /// \param timeout_ms Optional timeout for API call, in microseconds, the
+  /// request is allowed to take. \return Error object indicating success or
```

Comment:
Formatter seem to have introduced a skewed up documentation. \return should be in a new line.